reactnative-todo/src/components/utilities/request.js

```diff
@@ -26,7 +24,8 @@ async function request(method, resource = '', body = null) {
     const json = await FRAuthSampleBridge.getAccessToken();
     const tokens = JSON.parse(json);
     const { tokenType, value } = tokens;
-    const res = await fetch(`${API_URL}/todos/${resource}`, {
+     // edit the url here in fetch with the url for your server
+    const res = await fetch(`https://api.example.com:9443/todos/${resource}`, {
```

Comment:
Do we have a `constants` file or a better place to put such a value. I'm afraid this is a bit hidden. Also, I'd like to have a check when the app boots up to check if a value is present, if not, it errors.